Emby.Naming/Video/ExtraRuleType.cs

```diff
@@ -10,12 +10,12 @@ public enum ExtraRuleType
         Suffix = 0,
 
         /// <summary>
-        /// Match <see cref="ExtraRule.Token"/> against the file name.
+        /// Match <see cref="ExtraRule.Token"/> against the file name, excluding the file extension.
         /// </summary>
         Filename = 1,
 
         /// <summary>
-        /// Match <see cref="ExtraRule.Token"/> against the a regex.
+        /// Match <see cref="ExtraRule.Token"/> against the file name, including the file extension.
```

Comment:
Are you sure how it works? I haven't checked the code personally, so just asking...

Reply:
Yeah, the file name is extracted [here](https://github.com/jellyfin/jellyfin/blob/master/Emby.Naming/Video/ExtraResolver.cs#L73) using `Path.GetFileName()` which is [documented](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getfilename?view=netcore-3.1#System_IO_Path_GetFileName_System_String_) as 

> Returns the file name and extension of the specified path string.